src/Microsoft.Tye.Hosting/ProcessRunner.cs

```diff
@@ -51,12 +53,32 @@ private async Task BuildAndRunProjects(Application application)
 
                 string path;
                 string args;
+                var buildArgs = string.Empty;
                 string workingDirectory;
                 if (serviceDescription.RunInfo is ProjectRunInfo project)
                 {
                     path = project.RunCommand;
                     workingDirectory = project.ProjectFile.Directory.FullName;
                     args = project.Args == null ? project.RunArguments : project.RunArguments + " " + project.Args;
+
+                    foreach (var supportedProperty in BuildPropertiesToOptionsMap)
```

Comment:
Why do you need this?

Reply:
Just removed, I thought that some properties must be turned into command option. But is not required.